Off-Top-App/off-top-flutter
# OT-AccountCreation | Added a new feature where the user can signup for Off-Top

**What I did:**
- Developed a way where the user can signup for Off-Top if they are new. Providing a new signup page where they can enter their information.

**How to test:**

- Make sure Zookeeper, Kafka, Spring-boot, and port forwarding is running to log into app.
Have mySQL running for communication with database. You can use MySQL Workbench to view your data.

- When signing for google make sure you use an email that is not in your database.
![image](https://user-images.githubusercontent.com/38572314/85369487-0c49e780-b4e2-11ea-988f-423af6c97c85.png)

- Once you completed the appropriate fields click the submit button.
![image](https://user-images.githubusercontent.com/38572314/85370223-394aca00-b4e3-11ea-9af0-3ab520d46c30.png)

- You will see this when data was successfully submitted.
![image](https://user-images.githubusercontent.com/38572314/85370366-7020e000-b4e3-11ea-98b1-dd4f2f9f7c58.png)

- If everything is good to go you will see that your DB has changed with the data you entered. If not, refresh the DB.
![image](https://user-images.githubusercontent.com/38572314/85370031-f852b580-b4e2-11ea-871c-eeb16d409311.png)

- Click sign in with google and you will be taken to the main page.
